### ISSUE TITLE
WordPress Admin Posts

### ISSUE THREAD

**Issue by feel-brannoo** [herval/antisocial-chrome#1]
Hello! I just recently left a review for your extension. Great work!

I am noticing a potential issue with your AS logo not displaying as intended, I'm assuming, in the table header of the WordPress (currently using v4.5.2) Posts page. I'm including a couple screenshots of the header area. One is of it in the current state, another is after I tried editing the CSS with the Developer Tool.

I believe the issue is with this code:

<img width="1069" alt="screen shot 2016-06-12 at 6 02 36 pm" src="https://cloud.githubusercontent.com/assets/9667471/15995064/1354e9e0-30c9-11e6-8e7e-72c1cfbe7ca2.png">

`img.emoji {
    display: inline!important;
    border: none!important;
    height: 1em!important;
    width: 1em!important;
    margin: 0 .07em!important;
    vertical-align: -.1em!important;
    background: 0 0!important;
    padding: 0!important;
    -webkit-box-shadow: none!important;
    box-shadow: none!important;
}`

I tried changing the **em** units to **px**:

<img width="1073" alt="screen shot 2016-06-12 at 6 02 10 pm" src="https://cloud.githubusercontent.com/assets/9667471/15995065/13556136-30c9-11e6-98a0-e07b04fb86fd.png">

`img.emoji {
    display: inline!important;
    border: none!important;
    height: 16px!important;
    width: 16px!important;
    margin: 0 1px!important;
    vertical-align: 10px!important;
    background: 0 0!important;
    padding: 0!important;
    -webkit-box-shadow: none!important;
    box-shadow: none!important;
}`

Perhaps this is a non-issue, but I thought this might help. Thanks for the extension!

**Comment by herval** [comment]
interesting :-) the current font size is actually set to 6em (which is around 96px), so the icons do show up as blown out (kind of on purpose). Making it 16px would make it very small on most sites... :(

**Comment by feel-brannoo** [comment]
I see the 96px icons in place of the comment sections, which is great. In that context, it makes perfect sense to make them large. However, it seems a bit large to me in the WordPress backend. It seemed the width and height was set to **1em!important**, as I copied from the source code. But maybe Chrome isn't honoring the **em** units, so I tried using a unit of **px** and it seemed to fix the "issue."

But if it's meant to be 96px in that area, too, then it it's working fine. I just thought it seemed a little large for backend display. Anyways, thanks!
